src/components/Modal/ModalProducts.js

```diff
@@ -100,11 +101,17 @@ function ModalProducts({
       onClose={() => setOpenModal(false)}
     >
       <DialogTitle style={{ paddingBottom: "0px" }}>
-        <Grid container>
+        <Grid container style={{ display: "flex" }} justify="space-between">
```

Comment:
grids are always display flex iirc

Reply:
Oh really? It wasn't working without that.

Reply:
![image](https://user-images.githubusercontent.com/8600507/99114882-88db8080-25bf-11eb-9f9f-21a946bda9df.png)
once you specify container within a grid, display flex will be set

Reply:
Yeah, I followed that, that's why I added flex. LOL, Maybe it wasn't working at first coz the size 5 and 7, so it just made it fit. It just worked when I changed it to 5 and 5 and I didn't notice it XD

Reply:
💯